### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by mingwli**
1. merge tests result [ref](https://github.com/microsoft/azure-pipelines-tasks/issues/10353#issuecomment-668819166)
image
~~2. no need to checkout in regression tests~~ `git` command will be inaccessible if checkout set to none

### CODE REVIEW

**Review by herohua — APPROVED** [review]

File: azure-pipelines-v3.yml [hunk 1]
```diff
@@ -41,12 +41,18 @@ jobs:
...
-  
+
+  - task: reportgenerator@4
+    displayName: merge test results
+    inputs:
+      reports: test/**/coverage.cobertura.xml
+      targetdir: $(Pipeline.Workspace)/coveragereport
+
   - task: PublishCodeCoverageResults@1
     displayName: uplodoad code coverage result
```
Comment (yufeih) [inline azure-pipelines-v3.yml:52]:
```suggestion
    displayName: Upload code coverage result
```

**Review by yufeih — APPROVED** [review]
